src/CommunityToolkit.Maui.MediaElement/CommunityToolkit.Maui.MediaElement.csproj

```diff
@@ -63,6 +69,12 @@
     <PackageReference Include="Xam.Plugins.Android.ExoPlayer.Transformer" Version="[2.19.1,)" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiPackageVersion)"/>
+    <PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiPackageVersion)" />
+  </ItemGroup>
+  <ItemGroup>
+    <MauiImage Include="Resources\whitefs.png" BaseSize="36,36" Resize="true" pack="true" PackagePath="." />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Extensions\" />
   </ItemGroup>
```

Comment:
this can be removed 
```suggestion
```

Reply:
Thanks @pictos for the recommendation. It has been applied. Marked as resolved.